Documentation/building/unix-instructions.md

```diff
@@ -59,3 +59,19 @@ If you see errors along the lines of `SendFailure (Error writing headers)` you m
 ```sh
 mozroots --import --sync
 ```
+
+## Steps to Build on Ubuntu
+
+*Note: verified on Ubuntu 14.04 LTS*
+
+1. Install git `sudo apt-get install git`
+2. Clone the corefx repo `git clone https://github.com/dotnet/corefx.git`
+3. Install curl-dev `sudo apt-get install libcurl4-openssl-dev`
```

Comment:
This version looks good. Thanks for appeasing my nit-picking :smile: . One more thing: Could you take a look at the discussion down below regarding clang 3.5 vs. clang 3.6? If clang 3.5 is available on Ubuntu 14.04, could you switch the instructions to use that, and remove the workaround involving the symlink? It would be simpler to just install clang 3.5 and have it work immediately than involving an extra step of linking if its not necessary.